Automattic/sensei
# Try Playwright

### Changes proposed in this Pull Request

* Add the Playwright E2E testing framework. Props to https://github.com/WordPress/gutenberg/pull/38570.

### Testing instructions

* Have Docker Desktop / [Colima](https://github.com/abiosoft/colima).
* Run `npm install`.
* Run `npx playwright install`
* Run `npm run test:e2e-playwright` or `npm run test:e2e-playwright -- --headed` if you want to see the browser.
* The `example.spec.js` tests should be passing.
* E2E tests should be passing in CI.

### TODO

* Integrate Gutenberg's [e2e-test-utils-playwright](https://github.com/WordPress/gutenberg/tree/trunk/packages/e2e-test-utils-playwright).
* Convert the existing E2E tests.
* Tweak the `playwright.config.js` config if needed.
* Tweak the  `e2e-playwright.yml` workflow if needed.
